modules/ROOT/pages/hardware-and-software-requirements.adoc

```diff
@@ -1,40 +1,61 @@
-= Hardware and Software Requirements
+= Mule Runtime Requirements for Hardware and Software
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 :keywords: mule, requirements, jdk, installation
 
-Recommended minimum hardware for running Mule Runtime:
+== Minimum Hardware Requirements
+
+Mule Runtime (Mule) requires the following minimum hardware resources:
 
 * 2GHz CPU or 1 virtual CPU in virtualized environments
 * 1GB of RAM
 * 4GB of storage
 
 Adjust RAM to match your latency requirements and the size and number of simultaneous messages that applications process.
 
-Software requirements are:
+== Minimum Software Requirements
+
+Mule requires the following minimum software resources:
 
-* One of the following Java versions:
+* One of the following supported Java versions:
 +
-** Oracle JDK 1.8.0 (JDK 1.8.0_161 recommended)
+** Oracle JDK 1.8.0 (JDK 1.8.0_161 recommended). 
 +
 [NOTE]
---
-If you are planning to run a Standalone instance of Mule Runtime Anypoint Studio freezes during start up if your installed JDK runtime is 1.8 Update 152 (1.8.0_152). It is recommended that you downgrade back to 1.8.0_151.
---
+If you run a Standalone instance of Mule and Anypoint Studio freezes during start up, check to see if the JDK where Studio is installed is 1.8 Update 152 (1.8.0_152). If it is, downgrade to 1.8.0_151 to correct the problem.
++
 ** IBM JDK 1.8
 ** OpenJDK 8 (JDK 1.8.0_162 recommended)
 +
-* For Microsoft Dynamics GP connector install Java Cryptography Extension (JCE) 8 after installing JRE or JDK.
-
-Mule Runtime runs as a standalone server on operating systems that support one of the required Java versions. An application server is not required; however, Mule Runtime can run under one. +
-Mule Runtime does not require a database unless you need to access a data store.
-
-[%header,cols="20a,75a"]
-|===
-|Software |Version
-|Operating systems |MacOS 10.11.4, HP-UX 11i V3, AIX 7.2, Windows 2012 R2 Server, Windows 8.1, Solaris 11.3, RHEL 7, Ubuntu Server 15.04 and 16.04, any other Linux distribution which runs a supported JDK and Linux kernel 3.13+
-|Application servers |Tomcat 7 and 8, WebLogic 12c, WebSphere 8, WildFly 8 and 9, Jetty 8 and 9
-|Databases |Oracle 11g, Oracle 12c, MySQL 5.5+, IBM DB2 10, PostgreSQL 9, Derby 10, Microsoft SQL Server 2014
-|Browsers for working on Anypoint Platform | Firefox or Chrome (latest), Internet Explorer 10 or later; minimum screen resolution of 1024x768
-|===
\ No newline at end of file
+* An operating system that supports one of the following Java versions:
++
+** MacOS 10.11.4
+** HP-UX 11i V3
+** AIX 7.2
+** Windows 2016, Windows 2012 R2 Server, or Windows 8.1
```

Comment:
Alejandro Nosenzo says that Windows 2016 Server / Windows 10 are recommended. You should check with him about this part: "Windows 2012 R2 Server, or Windows 8.1"